### PR TITLE
NUKED write_reload_steps config value!

### PR DESCRIPTION
**Pull request by Christian-B**
https://github.com/SpiNNakerManchester/SpiNNFrontEndCommon/issues/396

### CODE REVIEW

File: spinn_front_end_common/interface/abstract_spinnaker_base.py [hunk 1]
```diff
@@ -596,10 +596,9 @@ def _load_config(self, filename, defaults, validation_cfg):
 
     # options names are all lower without _ inside config
     DEBUG_ENABLE_OPTS = frozenset([
-        "reportsenabled", "displayalgorithmtimings",
```
Comment (dkfellows) [inline spinn_front_end_common/interface/abstract_spinnaker_base.py]: What's going on here?

Comment (Christian-B) [reply spinn_front_end_common/interface/abstract_spinnaker_base.py]: When the user sets mode = debug ASB (or now ConfigHandler) looks for config settings to change from false to true.

Similarly if reportsEnabled=False it looks for ones to set false.

The way this is done is to either
1. Look if the config name has "write" in it
2. The config name is in this list.

On a side note DEBUG_ENABLE_OPTS should be moved to ConfigHandler

Comment (dkfellows) [reply spinn_front_end_common/interface/abstract_spinnaker_base.py]: So (the equivalent of) `displayalgorithmtimings` has moved to config handler? That's fine.

Comment (Christian-B) [reply spinn_front_end_common/interface/abstract_spinnaker_base.py]: No in this case the config value "displayalgorithmtimings" was no longer used anywhere.

So what was the reason for setting it to true other than confusion!

**Pull request merged**